library/Falcon/Connector/WordPress.php

```diff
@@ -505,13 +505,16 @@ public function get_comment_subscribers( WP_Comment $comment ) {
 			);
 		}
 
-
 		$users = get_users( $query );
 
+		// Also get the original post author, they should always be notified.
```

Comment:
Should this always be the case, or should it only apply if they've selected "participant"? I lean towards the latter, but interested to know whether this was an intentional choice.

Reply:
I was leaning towards always getting a notification, regardless of whether or not they've marked "Participant". If you've posted a post, it is very unlikely that you'd not want to get notifications. I'm happy to change this assumption, however.

Reply:
@rmccue  i've thought about this more and honestly i think if you author a post, regardless if you have set comments to participants or not, you should get comments and responses. 

People don't often check what the settings are.

Reply:
My 2 cents. This should be a default; with a supporting action for authors to 'unwatch' or 'leave a thread'...  particularly important if a thread is particularly **hot or popular** and the author wants to remove themselves from the discussion; we should respect the intention of the individual.

Personally, I am also okay with that suggestion being moved to 'forward work' and accepting the new default notification proposed here. GGWP @mikeselander